tests/abc_test.py

```diff
@@ -23,8 +23,10 @@ def test_mcabc_inference_on_linear_gaussian(
     sass_expansion_degree=1,
     kde=False,
     kde_bandwidth="cv",
+    num_iid_samples=1,
```

Comment:
please add test cases above for `num_iid_sample>1` and for the new distances. We need to make sure they work accurately.

Reply:
solved.